### DISCUSSION TITLE
velero restic annotation for namespace instead of Pod

### DISCUSSION THREAD

**Discussion by myvelmurugan** [vmware-tanzu/velero#4293] [Community support Q&A]
Hi,

We use velero with restic to take PVC backup for Oracle Kubernetes Engine  (Oracle cloud). Since its not supporting any snapshot plugin we use restic.

Now we do annotate pods (Opt-in) method with backup volume name. Is there any way to annotate for entire namespace so that pods which has PVC should automatically take backup?

**Comment by sseago** [comment]
There isn't currently a namespace-level annotation for restic, but there is the default-volumes-to-restic flag on backup creation if you want to do this for all namespaces in your backup and then you wouldn't need to annotate each pod: https://velero.io/docs/main/restic/#using-the-opt-out-approach

If you only want this for one namespace, you could consider a namespace-scoped backup for just this namespace.
